### PR TITLE
Update LFF converter to latest FontoBene specifications

### PR DESCRIPTION
**Pull request by ubruhin**
See commit messages for details.

### CODE REVIEW

**Comment by ubruhin** [comment]
Oh, I forgot to implement https://github.com/fontobene/fontobene/pull/28 :scream: 

EDIT: Fixed.

File: lff/convert.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def convert_codepoint(match):
     # Match regexes
     arc_re = re.compile(r',A-?[0-9\.]+')
     ref_re = re.compile(r'C([0-9a-fA-F]{4,6})')
-    metadata_string_re = re.compile(r'#\s*([a-zA-Z0-9]*):\s+(.+)')
+    metadata_string_re = re.compile(r'#\s*([a-zA-Z0-9 ]*):\s+(.+)')
```
Comment (dbrgn) [inline lff/convert.py]: Maybe use `\s` in the regex instead of the space to catch all kinds of spaces?

File: lff/convert.py [hunk 2]
```diff
@@ -114,6 +114,9 @@ def split_oneliner(match):
             font_author = metadata.get('author', metadata.get('creator', 'converted'))
             print('author = {}'.format(font_author))
             print('license = {}'.format(metadata.get('license', 'unknown')))
+            print('letter_spacing = {:g}'.format(float(metadata.get('LetterSpacing', 3))))
+            print('word_spacing = {:g}'.format(float(metadata.get('WordSpacing', 6.75))))
+            print('line_spacing = {:g}'.format(float(metadata.get('LineSpacingFactor', 1)) * 15.0))
```
Comment (dbrgn) [inline lff/convert.py]: Did we discuss those keys already?

**Review by dbrgn — COMMENTED** [review]
Great thanks!

Did you test a few files with this converter?

File: lff/convert.py (hunk 2 above)
Comment (ubruhin) [reply lff/convert.py]: Not yet, but any feedback is welcome :wink:  https://github.com/fontobene/fontobene/pull/29

**Comment by ubruhin** [comment]
> Did you test a few files with this converter?

So far I tested only the `unicode.lff` file.

File: lff/convert.py (hunk 1 above)
Comment (ubruhin) [reply lff/convert.py]: Done.

**Review by dbrgn — APPROVED** [review]

**Pull request merged**